test/e2e/testnet/defaults.go

```diff
@@ -4,7 +4,7 @@ var DefaultResources = Resources{
 	MemoryRequest: "400Mi",
 	MemoryLimit:   "400Mi",
 	CPU:           "300m",
-	Volume:        "1Gi",
+	Volume:        "0Gi",
```

Comment:
The minimum size for a volume is `1Gi`.
If you don't want a volume, you should not call `AddVolume` for that Instance.